aws_lambda_powertools/utilities/feature_toggles/schema.py

```diff
@@ -26,59 +26,69 @@ class SchemaValidator:
     def __init__(self, logger: Logger):
         self._logger = logger
```

Comment:
@heitorlessa - probably don't need the logger anymore?

Reply:
probably not. I'll look at the schema last, as there are opportunities to make it easier to maintain